src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -535,7 +535,8 @@ let setup_daemon logger =
   and simplified_node_stats =
     flag "--simplified-node-stats"
       ~aliases:[ "simplified-node-stats" ]
-      no_arg ~doc:"whether to report simplified node stats (default: false)"
+      (optional_with_default true bool)
```

Comment:
no? wrong PR?

Reply:
It came from the merge through of the `compatible` PR :man_shrugging:

Reply:
👍🏼